### PR TITLE
Fix responsive issue on WelcomeBox

### PR DESCRIPTION
**Pull request by iamstratos**
Associated Issue: #3999

### Summary of Changes

* Changed WelcomeBox divs in order to look better on responsive

### Screenshots/Videos
WelcomeBox Responsive

### CODE REVIEW

**Comment by jasonLaster** [comment]
Thanks for the PR!

I like the responsive look. Can we keep the regular UI though when it is not minified?

**Comment by iamstratos** [comment]
@jasonLaster Sure, I'll commit later :)

**Comment by codehag** [comment]
@iamstratos Hey! just wanted to check where you are with this! Thanks for the work :)

**Comment by iamstratos** [comment]
@codehag I have this ready but I cannot run the tests with `yarn test` or `yarn test-all`.

I get this error
```
$ jest
s"  was unexpected at this time.
```

**Comment by jasonLaster** [comment]
@iamstratos mind pushing with `--no-verify`? we can sort it out here

**Comment by iamstratos** [comment]
@jasonLaster Done!

**Comment by jasonLaster** [comment]
now, i'm curious :) what's mobile-layout? 

mind sharing some screeshots?

what was the issue w/ jest?

**Comment by iamstratos** [comment]
@jasonLaster Nah, my bad. Just realized I've used the word "mobile" even if I didn't mean to.
"normal-layout" is the one that there was before I create mine, which is "mobile-layout" (screenshot on the first post here).
Gonna change the name though, my bad.

File: src/components/WelcomeBox.js [hunk 1]
```diff
@@ -70,7 +83,6 @@ class WelcomeBox extends Component {
               {searchProjectLabel}
             </p>
           </div>
-          {this.renderToggleButton()}
```
Comment (codehag) [inline src/components/WelcomeBox.js]: do we maybe still want this in the normal layout?

Comment (jasonLaster) [inline src/components/WelcomeBox.js]: brilliant!

**Pull request merged**